addon/services/drag-coordinator.js

```diff
@@ -5,9 +5,13 @@ import { isEqual } from '@ember/utils';
 
 
 function indexOf(items, a) {
-  return items.findIndex(function (element) {
-    return isEqual(element, a);
+  var returnIndex=null;
+  items.forEach(function(element, index) {
```

Comment:
@ianpetzer This seems ok, but it doesn't break when it finds an item like the findIndex does.  If you use a "for in" you can break sooner.  That will make for more performant code.

https://stackoverflow.com/questions/2641347/short-circuit-array-foreach-like-calling-break